pkg/render/node.go

```diff
@@ -67,6 +72,7 @@ func Node(
 	nodeAppArmorProfile string,
 	clusterDomain string,
 	nodeReporterMetricsPort int,
+	bgpLayoutHash string,
```

Comment:
This feels odd to me to pass in the hash. I think usually the component that needs a hash is calculating it but there is no particular reason for that. So I think passing this in is fine. I just wanted to mention my take in case you had an reservations about passing the hash.
tl;dr, this is not a request to change anything.

Reply:
I did think about this and went through a few iterations before arriving here.  I think this works well, even though it's a different place than other rmeta.AnnotationHash calls.